src/inmanta/app.py

```diff
@@ -767,9 +776,7 @@ def _get_watched_file_handler(options: argparse.Namespace) -> logging.handlers.W
     return file_handler
 
 
-def _convert_to_log_level(level: int) -> int:
-    if level >= len(log_levels):
-        level = len(log_levels) - 1
```

Comment:
This check made sure that we wouldn't get an exception when somebody passed in `-vvvvvvvv` on the commandline. I think we should maintain that behavior.

Reply:
I put it in the `_convert_cli_log_level` function. I think it is ok?

Reply:
That's not enough. This method is used by the `_get_watched_file_handler()` method. This new implementation crashes when a log level is passed in above the maximum log level while the previous implementation wouldn't. This might cause the orchestrator to crash after an upgrade. We should prevent that.